src/components/publishing/sections/fullscreen_viewer/__test__/fullscreen_viewer.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { shallow } from "enzyme"
+import "jest-styled-components"
+import * as React from "react"
+import * as renderer from "react-test-renderer"
+import { Images } from "../../../fixtures/components"
+import FullscreenViewer from "../fullscreen_viewer"
+
+jest.mock("react-slick", () => {
+  const React = require("react")
+  return props => <div>{props.children}</div>
+})
+it("renders properly", () => {
+  const onClose = jest.fn()
+  const viewer = renderer.create(<FullscreenViewer images={Images} show onClose={onClose} />).toJSON()
+  expect(viewer).toMatchSnapshot()
+})
+
+it("closes the viewer on ESC keydown", () => {
+  const onClose = jest.fn()
+  const viewer = shallow(<FullscreenViewer images={Images} show onClose={onClose} />)
+  viewer.simulate("keyDown", { keyCode: 27 })
+  expect(onClose.mock.calls.length).toBeGreaterThan(0)
+})
```

Comment:
nice tests

Reply:
@kanaabe you can change that to `expect(onClose).toBeCalled()`

Reply:
Or `toBeCalledWith` also - https://facebook.github.io/jest/docs/en/expect.html#tohavebeencalled